### PR TITLE
[NEXT IN MERGE QUEUE AFTER BUG] feat: LT-1363 - Specific Countries For Open Goods

### PR DESCRIPTION
**Pull request by cstobart**

### CODE REVIEW

File: templates/apply_for_a_licence/goodstype/countries.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <span class="govuk-heading-l">Explain where each item is going (Optional)</span>
+    </div>
+
+    <p class="govuk-caption-m">
+        <a class="govuk-link" href="{% url 'apply_for_a_licence:overview' draft_id %}">
+            Skip this section
+        </a> if you are sending every item to every country</p>
+
+    <form method="post">
+
```
Comment (BenAndrew-Kainos) [inline templates/apply_for_a_licence/goodstype/countries.html]: quite a lot of empty newlines (maybe not needed)

File: templates/apply_for_a_licence/goodstype/countries.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{% extends "layouts/base.html" %}
+
+{% block back_link %}
+    <a href="{% url 'apply_for_a_licence:overview' draft_id %}" class="govuk-back-link">Back to Application</a>
+{% endblock %}
+
+{% block body %}
+
+    <div>
```
Comment (jfaracik) [inline templates/apply_for_a_licence/goodstype/countries.html]: Get rid of div, replace span with h2

File: templates/apply_for_a_licence/goodstype/countries.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            <a class="govuk-link" href="{% url 'apply_for_a_licence:overview' draft_id %}">Skip this section</a> if you are sending every item to every country
+        </p>
+
+    </form>
+
+{% endblock %}
+
+{% block javascript %}
+    <script>
+        function selectAll(value) {
```
Comment (jfaracik) [inline templates/apply_for_a_licence/goodstype/countries.html]: Named incorrectly, excessive selector

File: ui_automation_tests/pages/goods_countries_page.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+                element.click()
+
+    def deselect_all(self):
+        elements = self.driver.find_elements_by_css_selector(".govuk-checkboxes__input")
+        for element in elements:
+            if element.is_selected():
+                element.click()
+
+    def all_selected(self):
+        elements = self.driver.find_elements_by_css_selector(".govuk-checkboxes__input")
```
Comment (jfaracik) [inline ui_automation_tests/pages/goods_countries_page.py]: Duplicated code

**Pull request merged**